QtCollider/QcApplication.cpp

```diff
@@ -181,12 +181,27 @@ bool QcApplication::notify(QObject* object, QEvent* event) {
     bool result = QApplication::notify(object, event);
 
 #ifdef Q_OS_MAC
+    bool isCloseEvent = false;
+    bool isKeyEvent = false;
+    if (event->type() == QEvent::KeyPress) {
+        QKeyEvent* kevent = static_cast<QKeyEvent*>(event);
```

Comment:
it's better to use `auto` when it doesn't obscure the type, and especially when the type is already known from the right hand side of an initialization (for example a cast or obvious function name). this keeps code concise and makes it easier to read and maintain.